lms/envs/test.py

```diff
@@ -588,3 +599,6 @@
 COMPREHENSIVE_THEME_LOCALE_PATHS = [REPO_ROOT / "themes/conf/locale", ]
 
 LMS_ROOT_URL = "http://localhost:8000"
+NODEBB_RETRY_DELAY = 60
+NODEBB_ENDPOINT = "http://local.philanthropyu.org:4567"
+NODEBB_MASTER_TOKEN = "b4f6c9c3-d778-4b47-bdbc-73535cfdbe3b"
```

Comment:
import it from auth file

Reply:
Importing from auth file is non-trivial and is unnecessary in this case. I have supplied a hard coded token from our custom philu devstack box. That should suffice for any future team members.